plugins/search-backend-module-elasticsearch/config.d.ts

```diff
@@ -21,6 +21,10 @@ export interface Config {
      * Options for ElasticSearch
      */
     elasticsearch?: {
+      /**
+       * Prefix to be used for all index creation. Value will be put in front of the index as is.
+       */
+      indexPrefix?: string;
```

Comment:
Can we also document this here: https://github.com/backstage/backstage/blob/master/docs/features/search/search-engines.md